mlflow/R/mlflow/document.R

```diff
@@ -3,7 +3,7 @@ Rd2md::ReferenceManual()
 
 # Remove markdown package description
 markdown_doc <- readLines("Reference_Manual_mlflow.md")
-first_function <- which(grepl("crate", markdown_doc))[[1]]
+first_function <- which(grepl("mlflow_client", markdown_doc))[[1]]
```

Comment:
I don't really have a context here but it seems a bit suspicious. Is this hardcoded so that it finds the first function that's documented?

Reply:
Yeah it is...https://github.com/mlflow/mlflow/pull/1256 contains a more robust fix for this (but still not great). In particular this code is trying to strip out a header delimited by triple-backticks from `Reference_Manual_mlflow.md`, so the linked code uses the triple-backtick delimiter to identify & strip the header. This should at least be more stable than trying to filter to the first API based on alphabetical ordering.

I'll copy the logic from #1256 into this PR for now